server/wfprev-api/src/main/java/ca/bc/gov/nrs/wfprev/data/entities/FundingSourceCodeEntity.java

```diff
@@ -0,0 +1,56 @@
+package ca.bc.gov.nrs.wfprev.data.entities;
+
+import com.fasterxml.jackson.annotation.JsonIgnoreProperties;
+import jakarta.persistence.*;
+import lombok.*;
```

Comment:
I think we want to avoid using wildcard imports. It's more efficient and readable to just import the classes that are needed

Reply:
seconded. We shouldn't use wildcards on imports. Put the full import in place.

Reply:
Thanks, updated to use full individual imports